scripts/app.js

```diff
@@ -30,7 +28,7 @@
       $.getJSON('/data/projects.json', function(rawData){
         Project.loadAll(rawData);
         localStorage.setItem('rawData', JSON.stringify(rawData));
-        ProjectView.initIndexPage();
+        projectView.initIndexPage();
```

Comment:
You should replace this line with the line similar to the one in your if block that calls the function that you pass in.